esy-solve/NpmRegistry.re

```diff
@@ -49,8 +49,16 @@ let rec retryInCaseOfError = (~num, ~desc, f) =>
 let make = (~concurrency=40, ~url=?, ()) => {
   let url =
     switch (url) {
-    | None => "http://registry.npmjs.org/"
-    | Some(url) => url
+    | None => "http://registry.npmjs.org"
+    | Some(url) =>
+      let reverse = url |> String.to_seq |> List.of_seq |> List.rev;
+      let lastChar = reverse |> List.hd;
+
+      if (lastChar == '/') {
+        reverse |> List.tl |> List.rev |> List.to_seq |> String.of_seq;
+      } else {
+        url;
+      };
```

Comment:
I'm not sure that is the right solution, but if we're going to go with this "workaround", using `String.sub` and `String.length` is a better idea

Reply:
`String.to_seq |> List.of_seq` seems a bit of an overkill I agree. All we really need to inspect is the trailing character in the url string.